src/containers/pages/FocusInvestigation/map/active/index.tsx

```diff
@@ -306,7 +306,9 @@ class SingleActiveFIMap extends React.Component<
             <Col xs="8">
               <h2 className="page-title mt-4 mb-4">
                 {plan && plan.plan_title} {INVESTIGATION}{' '}
-                <h5 style={{ display: 'inline' }}>{statusBadge}</h5>
+                <p className="h5" style={{ display: 'inline' }}>
```

Comment:
I am not feeling great as having "h5" as the CSS class.  It seems an odd choice to me

Reply:
Alternatives:
- Using a custom css class to make the `p` look like h5.
- use a h3 for proper nesting:
    - leave the default styling for the h3
    - write a custom style class to make it look like h5 .
- others?
I think the domElement in question has to be a `h3` or any other text non-heading domElement.

Reply:
My question was more regarding the choice of name "h5" i.e. it feels weird to have it named as the name of another html element

Reply:
the `h5` is from bootstrap.

Reply:
aaah, okay